### PR TITLE
[pulsar-client-tools] Add resource close in CmdBrokerStats and some minor fixes

### PR DESCRIPTION
**Pull request by murong00**
### Motivation

Resources in `CmdBrokerStats` should be closed.

### Modifications

Use `try-with-resources` and add description for `load-report` in admin-cli doc.

### CODE REVIEW

File: pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdBrokerStats.java [hunk 1]
```diff
@@ -139,8 +144,4 @@ public CmdBrokerStats(PulsarAdmin admin) {
         jcommander.addCommand("load-report", new CmdLoadReport());
     }
 
-    public void addCommands(String commandName, CliCommand cliCommand) {
```
Comment (jiazhai) [inline pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdBrokerStats.java]: There is no one using this?

Comment (murong00) [reply pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdBrokerStats.java]: yeah, idea shows this method is never used.

File: pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdBrokerStats.java [hunk 2]
```diff
@@ -139,8 +144,4 @@ public CmdBrokerStats(PulsarAdmin admin) {
         jcommander.addCommand("load-report", new CmdLoadReport());
```
Comment (jiazhai) [inline pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdBrokerStats.java]: I see. the documents are not aligned with this part of code. 👍

**Comment by merlimat** [comment]
run cpp tests

**Pull request merged**